src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -603,9 +646,8 @@ private static Properties loadProperties(File file)
     }
 
     /**
-     * This method creates in AuditListener an open stream for validation data, it must be closed by
-     * {@link RootModule} (default implementation is {@link Checker}) by calling
-     * {@link AuditListener#auditFinished(AuditEvent)}.
+     * Creates the audit listener.
+     *
```

Comment:
Method was not changed so why are we changing the javadoc?

Reply:
done